effects/data_base_effect.py

```diff
@@ -0,0 +1,71 @@
+from typing import List
+
+import message_text_config as config
+import user_interaction
+import utils
+from effects.effect import Effect
+from message_text_config import Errors
+
+
+class DataBase(Effect):
+    def __init__(self, game: Game, name: str, creator: Citizen) -> None:
+        super(DataBase, self).__init__(game, name, creator, 4)
+
+    def _activate_impl(self):
+        target_numbers = []
+        for _ in range(3):
+            target_number = user_interaction.read_index("config.type_target")
+            target_numbers.append(target_number)
```

Comment:
Please, use texts from message_text_config module. Insted of this string

Reply:
If there is no needed text - add it by yourself, based on Andrews pull request with requirements to the effects:
https://github.com/SurprisedBird/MurderBattleConsole/pull/14

Reply:
Same for other places, where messages are printed

Reply:
Ok, will do it

Reply:
Done